articles/iot-edge/how-to-deploy-cli-at-scale.md

```diff
@@ -150,6 +150,11 @@ Here's a basic layered deployment manifest with one module as an example:
   }
 }
 ```
+>[!NOTE]
+> Take note that this layered deployment manifest has a slightly different format than a standard deployment manifest. The desired properties of the runtime modules are collapsed using dot notation. This formatting is required for the Azure portal to recognize a layered deployment. For example:
+
+  - `properties.desired.modules.<module_name>`
+  - `properties.desired.routes.<route_name>`
```

Comment:
Thanks for going through a couple of iterations on these changes. I think this latest version looks great. Just one small formatting fix and then I can approve the PR. All of the lines of a "Note" box need the > symbol to format correctly, even if they're blank or indented: 

```suggestion
>[!NOTE]
> Take note that this layered deployment manifest has a slightly different format than a standard deployment manifest. The desired properties of the runtime modules are collapsed using dot notation. This formatting is required for the Azure portal to recognize a layered deployment. For example:
>
>  - `properties.desired.modules.<module_name>`
>  - `properties.desired.routes.<route_name>`
```

Reply:
Committed and looking forward to having it merged 😄 @kgremban